cosmwasm/contracts/global-accountant/.cargo/config

```diff
@@ -0,0 +1,3 @@
+[profile.release]
+lto = true
+strip = true
```

Comment:
I feel like this kind of thing should go in the workspace Cargo.toml rather than here.